### PR TITLE
Inhand Gun Sprites + Extra Gun Changes

### PR DESCRIPTION
**Pull request by SkowronX**
Essentially fixes https://github.com/d3athrow/vgstation13/issues/2187
- New inhand sprites for nearly all usable guns.
- Plasma weaponry is now deadlier.
- Plasma weaponry now use plasma research levels and plasma material. Because that makes a lot of sense right?
- Combat borg laser cannons use the correct laser cannon beam.
- Laser cannon sprite inconsistency corrected.
- Laser AK470s and Plasma MP40ks now visually display their ammo level.

New Plasma Guns:
plasmaguns

New Inhands:
inhands

### CODE REVIEW

File: code/modules/projectiles/projectile/energy.dm [hunk 1]
```diff
@@ -53,21 +53,42 @@
 
 /obj/item/projectile/energy/plasma
 	name = "plasma"
-	icon_state = "declone"
+	icon_state = "plasma"
+	var/knockdown_chance = 0
+	on_hit(var/atom/target, var/blocked = 0)
```
Comment (ComicIronic) [inline code/modules/projectiles/projectile/energy.dm]: PATHING

Comment (SkowronX) [inline code/modules/projectiles/projectile/energy.dm]: explain please

**Comment by ComicIronic** [comment]
All look very nice :+1:

File: code/modules/projectiles/projectile/energy.dm (hunk 1 above)
Comment (smet19) [inline code/modules/projectiles/projectile/energy.dm]: Replace `on_hit(var/atom/target, var/blocked = 0)` with `/obj/item/projectile/energy/plasma/on_hit(var/atom/target, var/blocked = 0)`

**Comment by Lokiamis** [comment]
Thank you! Very nice.

**Pull request merged**